FCCee/IDEA/compact/IDEA_o1_v03/DectDimensions_IDEA_o1_v02.xml

```diff
@@ -88,12 +88,18 @@
     <constant name="Inner_Vertex_half_length" value="257.5*mm"/>
     <constant name="Outer_Vertex_half_length" value="1050*mm"/>
 
-    <!-- Changing the radius of DCH is not enough to resize the detector, please contact the experts-->
-    <constant name="DCH_inner_cyl_R"       value=" 350 * mm "    />
-    <constant name="DCH_outer_cyl_R"       value=" 2000 * mm "    />
-    <constant name="DCH_L"                    value=" 4000 * mm "    />
-    <constant name="DCH_half_length_total"    value="2250*mm"/> <!-- this includes the services -->
-    
+    <!--  Drift Chamber parameters    -->
+    <!-- Changing the radius is not enough to resize the detector, please contact the experts-->
+    <!--    Parameters for the active volume (cylinder of gas)  -->
+    <constant name="DCH_inner_cyl_R"          value=" 350 * mm "   />
+    <constant name="DCH_outer_cyl_R"          value=" 2000 * mm "  />
+    <constant name="DCH_L"                    value=" 4000 * mm "  />
+    <constant name="DCH_Lhalf"                value=" DCH_L/2.0 "  />
+    <!--    Vessel cylinder surrounds the gas cylinder, eating 1mm extra in radius/z     -->
+    <constant name="DCH_vessel_thickness"     value=" 1 * mm"      />
+    <!--    Service layers fill the space from z=2m to z=2.25m  -->
+    <constant name="DCH_half_length_total"    value="2250*mm"      />
```

Comment:
Hi Alvaro, I would propose to keep only the global envelope in DectDimensions (this is what people working on other subdetectors need to know), the other parameters can be considered as "drift chamber internals" i.e. people working on the full IDEA implementation but not on the drift chamber do not need to know the vessel thickness, they just need to know where e.g. the silicon wrapper can start. I would therefore propose to keep only these in the DectDimension xml:

```
<constant name="DCH_inner_cyl_R" value=" 350 * mm " />
<constant name="DCH_outer_cyl_R" value=" 2000 * mm " />
<constant name="DCH_half_length_total" value="2250*mm" />
```

and e.g. the vessel thickness, service area, etc, (from DriftChamber xml) would "eat" space from these external dimensions.

What do you think?